### DISCUSSION TITLE
energy loss

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#661] [Q&A]
If we use energy training alone, only the energy information, no force information, how much error of training set and test set can be considered as the appropriate convergence?

**Comment by njzjz** [comment]
It depends on systems. It's better to plot error vs training steps to get more information.

**Comment by caolz** [comment]
This depends on your system. I train the gold cluster, and only train the energy to converge at 10-3 eV. When training energy and force, the energy converges by 10 -2 eV, and the force converges by 10-1 eV/A.
